src/emulator/host/include/host/state.h

```diff
@@ -17,6 +17,7 @@
 
 #pragma once
 
+#include "../../sfo.h"
```

Comment:
don't use relative include paths

Reply:
<sfo.h> doesn't work.

Reply:
we only use absolute paths: `<host/sfo.h>`

Reply:
sfo's not in host directory.

Reply:
yea nvm, have you tried adding emulator to CMakeLists.txt?

Reply:
Looks like SFO stuff needs to be moved to a new library.